JavaScript/JavaScriptSDK/Sender.ts

```diff
@@ -221,11 +221,21 @@ module Microsoft.ApplicationInsights {
          * xdr state changes
          */
         public static _xdrOnLoad(xdr: XDomainRequest, payload: string) {
-            if (xdr && (xdr.responseText + "" === "200" || xdr.responseText === "")) {
+            
```

Comment:
This is what https://dc.services.visualstudio.com/v2/track returns:
    {"itemsRecieved":2,"itemsAccepted":2,"errors":[]}
So, the response is never evaluated as successful